tst/regression/linear_modes.gold

```diff
@@ -1,1232 +1,1232 @@
--1.648375766769760290e-07
--1.613699532047422093e-06
--1.590226314757858155e-06
--1.027944017076012186e-08
-1.507039298405862370e-06
-1.656006947935212472e-06
-3.579589261123017248e-07
--1.261825516591687405e-06
--1.726366632561289537e-06
--6.925890308139091532e-07
-1.127179188628730195e-06
-1.738538725988569965e-06
-8.520018361738839483e-07
--8.333130985189662907e-07
--1.741587194395977850e-06
--1.147960233455665095e-06
-6.744110676648418459e-07
-1.741333821610902535e-06
-1.280954485934699437e-06
--3.385393181167661655e-07
--1.696443634018159185e-06
--1.504546304205754675e-06
--1.027943251450810224e-08
-1.613533829205230461e-06
-1.590252344359356449e-06
-1.850729367243258634e-07
--1.388583961626842159e-06
--1.700953865505401834e-06
--5.275213143991535331e-07
-1.261760805805104762e-06
-1.725819267574720927e-06
-6.926305500241761324e-07
--9.844303732986032653e-07
--1.740293455998816554e-06
--1.004289964934459536e-06
-8.332671041652599658e-07
-1.742274051589850055e-06
-1.148018616373521665e-06
--5.090030226700787340e-07
--1.734395104781504249e-06
--1.400634788150126604e-06
-1.648279423971163625e-07
-1.696371138759718979e-06
-1.504596158615509619e-06
-1.028067767113458230e-08
--1.507099273513728817e-06
--1.655912739855413748e-06
--3.579361695146866141e-07
-1.388528122621444856e-06
-1.701050240375294913e-06
-5.275539633029978847e-07
--1.127239214595986499e-06
--1.738200214381804411e-06
--8.519527517336309874e-07
-8.332671037666123795e-07
-1.740751701307312179e-06
-1.004344783992067638e-06
--6.744508687318256995e-07
--1.741251296447213591e-06
--1.280895004223554836e-06
-3.385188939962563274e-07
-1.734572544061627382e-06
-1.400691395214223112e-06
--1.648377751771490938e-07
--1.613654069756421429e-06
--1.590215504365368672e-06
--1.850607111848939276e-07
-1.507037537425401796e-06
-1.655958339317997208e-06
-3.579589518641616114e-07
--1.261822172097506300e-06
--1.725797783200190056e-06
--6.925890294727918293e-07
-9.843778963328664999e-07
-1.737178431780542629e-06
-8.520018413846791268e-07
--8.333139473196456831e-07
--1.742245021774007872e-06
--1.147960148770921662e-06
-5.089721369342301751e-07
-1.741310603667982871e-06
-1.280953932279122835e-06
--3.385393170266433916e-07
--1.696296137648279874e-06
--1.504548338331404144e-06
--1.027942829324316174e-08
-1.507035621867332381e-06
-1.590264345849646242e-06
-1.850729035477137626e-07
--1.388595280322372227e-06
--1.701050399214992493e-06
--5.275213135840591278e-07
-1.127179510866506852e-06
-1.726137462826043499e-06
-6.926305803623038528e-07
--9.844296559440050383e-07
--1.739797690169686256e-06
--1.004289941200076911e-06
-6.744111433083192837e-07
-1.741811333519652724e-06
-1.148018651135363127e-06
--5.090023546146911557e-07
--1.734527413667989552e-06
--1.400635785566348553e-06
-1.648279431176801968e-07
-1.613534133241710520e-06
-1.504597441069569458e-06
-1.028065202879326984e-08
--1.507090455995950591e-06
--1.655940491521968514e-06
--3.579361683397125794e-07
-1.261762431799203453e-06
+1.000005632783612652e+00
```

Comment:
Im not sure why this needed an upgold but still converges at 2nd order when I checked with `scripts/python/linmode.py`